README.md

```diff
@@ -158,29 +199,26 @@ Sample example:
 # tests/samples/sample.yaml
 project:
     meta:
-        name: project-sample
-        tags:
-          - example
-          - sample
-          - for
-          - testing
-          - purpose
+        name: my-project
+    foo: bar
+    list-items:
+        - item1
+        - item2
+        - item3
 ```
 
 Retrieve the project name:
 ```sh
 $ niet tests/samples/sample.yaml project.meta.name
```

Comment:
Sometimes we use /path/to/your/file.yaml sometimes tests/samples/sample.yaml.

Reply:
This section was introduced by the fact that we use the sample included in project sources.